datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JwsHeader.kt

```diff
@@ -24,12 +24,12 @@ import kotlinx.serialization.encodeToString
  */
 @Serializable
 data class JwsHeader(
-    @SerialName("alg")
-    val algorithm: JwsAlgorithm,
     @SerialName("kid")
     val keyId: String? = null,
     @SerialName("typ")
     val type: String? = null,
+    @SerialName("alg")
+    val algorithm: JwsAlgorithm,
```

Comment:
why was this moved?

Reply:
This is the ordering used in java. For some reason it was not compatible otherwise

Reply:
Versteh ich nicht, wenns funktioniert jetzt ... aber wo genau ist diese Order von Java vorgegeben?

Reply:
verstehn tu ichs auch nicht, aber ich kann mit sicherheit sagen, dass es so schon und anders nicht funktioniert hat. Grundsaetzlich ists ja egal.
Der Java constructor schaut so aus womit sie die order anscheinend implizit vorgeben

Reply:
da signaturen im input nicht permutationsinvariant sind wirds wsl sein dass java das eins nach dem andern runterschreibt und wenn wirs in einer anderen reihenfolge runterschrieben kommt halt was anders raus

Reply:
Das versteh ich, aber welchen Java Code meinst du, und hat das vorher nicht funktioniert?